proseco/acq.py

```diff
@@ -1033,7 +1042,11 @@ def calc_p_safe(self, verbose=False):
             p_n_cum = prob_n_acq(p_acqs)[1]  # This returns (p_n, p_n_cum)
 
             # Probability of 2 or fewer stars => conservative fail criteria
-            p2 = p_n_cum[2]
+            # handle the case of < 2 stars
+            if len(p_n_cum) <= 2:
```

Comment:
This is an unrelated fix.  I should pull it out of this PR and suggest it as its own PR.